R/classes.R

```diff
@@ -53,7 +44,14 @@ print.PartInv <- function(x, ...) {
   cat("Proportion selected: ", round(x$propsel, 3), "\n")
   cat("Cutpoint on the latent scale (xi): ", round(x$cutpt_xi, 3), "\n")
   cat("Cutpoint on the observed scale (Z): ", round(x$cutpt_z, 3), "\n")
-  cat("AI ratio: ", round(x$ai_ratio, 3), "\n\n")
+  #if(length(x$ai_ratio)==1) {
+  #  cat("AI ratio: ", round(x$ai_ratio, 3), "\n\n")
+  #} else {
+    cat("AI ratio: \n")
```

Comment:
Currently the output of an example is

```
AI ratio: 
   Focal_1
 0.7712291
```

Shall we make it a bit more informative, like "Adverse impact (AI) ratio for . . . "?

Reply:
Maybe we can also add a reference? Like "a result that is less than 80% is considered evidence of adverse impact"

Reply:
Perhaps it's just me, but I think a long description would be best placed in the documentation.

Reply:
Also, should we keep rounding in the results or not?